app/lib/actions/remote_execution/run_hosts_job.rb

```diff
@@ -65,6 +65,12 @@ def batch(from, size)
         hosts.offset(from).limit(size)
       end
 
+      # Will have default in ProxyBatchTriggering if we cant find provider
```

Comment:
When will it have a default? There seems to be a fallback to the global setting in base rex provider